typescript-loops-practice/find-index.ts

```diff
@@ -0,0 +1,11 @@
+/* exported findIndex */
+function findIndex(array: number[], value: number): number {
+  let answer = -1;
+  for (let i = 0; i < array.length; i++) {
+    answer++;
+    if (array[i] === value) {
+      return answer;
+    }
+  }
+  return -1;
```

Comment:
Why increment answer here? Instead of incrementing it, you could leave it at -1 and then return `answer` from the function.

OR other thing to think about, is `answer` really necessary here? Is there something else we could use that determines out `index` that we would return if the values match?